src/components/ProductList/ProductList.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useProduct } from '../../context/product-context'
+import { Productcard } from '../ProductCard/Productcard';
+import "../ProductList/ProductList.css";
+
+ export const ProductList = () => {
+     const {productList,setProductList}=useProduct();
+  return (
+    <div className="product-content-container">
+      <div className="grid-3-column-layout">
+          {productList.map((product)=>{
+              return <Productcard userProduct={product}/>
```

Comment:
I think you can name this prop  `products`
```
<ProductCard products={products} >

```
This is just a suggestion.

Reply:
will do this also